Automattic/jetpack
# Renovate-like: Update history and react-router deps

Attempt to update history and react-router deps

#### Changes proposed in this Pull Request:


* deps update

#### Testing instructions:





* run `yarn build`, make sure it works
* Launch JN site. click through & make sure it does not fails

#### Proposed changelog entry for your changes:


* none
